console/backend/src/main/java/nl/nn/adapterframework/management/web/TestPipeline.java

```diff
@@ -117,6 +117,8 @@ private String getPayload(Message<?> response) {
 		Object payload = response.getPayload();
 		if(payload instanceof String) {
 			return (String) payload;
+		} else if(payload instanceof byte[]) {
+			return new String((byte[]) payload);
```

Comment:
Technically it's possible however it's always UTF-8 and I plan on refactoring this bit of code soon. Since the new gateways It's become clear to me that we need to handle our responses better, instead of using a raw *T* type, and this ugly method..